packages-next/admin-ui/src/admin-meta-graphql.ts

```diff
@@ -152,8 +152,8 @@ type KeystoneAdminUIFieldMeta = {
   label: Scalars['String'];
   isOrderable: Scalars['Boolean'];
   fieldMeta: Maybe<Scalars['JSON']>;
-  viewsIndex: Scalars['Int'];
-  customViews: Maybe<Scalars['Int']>;
+  viewsIndex: Scalars['String'];
```

Comment:
This shouldn't be called viewsIndex if it's not an index and whatever the naming is, it should be consistent with customViews

Reply:
Yeah, I agree. I didn't want to include a change to this in this PR a) because there are already a lot of moving parts and b) I don't know what a good name would be 😅 If you have good ideas for consistent names I'm happy to ship a follow-up PR with the new and improved names 👍

Reply:
> I didn't want to include a change to this in this

I'd much prefer it all in one PR tbh.

`viewsHash`

Reply:
Cool, I can get on board with `viewsHash`, will make the change 👍 . Are we happy with `customViews`? Should it be `customViewsHash`?

Reply:
`customViewsHash`